### PR TITLE
added voice_cloning in jac speech

### PR DESCRIPTION
**Pull request by TharukaCkasthuri**

### CODE REVIEW

File: jaseci_ai_kit/jac_speech/jac_speech/vc_tts/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+soundfile<=0.11.0
+unidecode==1.3.6
+inflect<=6.0.2
+TTS==0.10.2
```
Comment (chandralegend) [inline jaseci_ai_kit/jac_speech/jac_speech/vc_tts/requirements.txt:5]: librosa==0.8.0

File: jaseci_ai_kit/jac_speech/jac_speech/stt/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 transformers>=4.25.1
-librosa>=0.9.2
+librosa<=0.9.2
```
Comment (chandralegend) [inline jaseci_ai_kit/jac_speech/jac_speech/stt/requirements.txt]: librosa>=0.8.0

File: jaseci_ai_kit/jac_speech/jac_speech/vc_tts/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+soundfile<=0.11.0
```
Comment (chandralegend) [inline jaseci_ai_kit/jac_speech/jac_speech/vc_tts/requirements.txt:1]: soundfile==0.11.0

File: jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+# **Text to Speech with Voice Cloning (`vc_tts`)**
+
+## **2. Synthesize audio using jaseci `vc_tts` module.**
+
+In this section we will take an input raw text and will output the list of amplitude values of the synthesized audio. This can use to save or play the audio files using a necessary library in a preffered language.
+
```
Comment (chandralegend) [inline jaseci_ai_kit/jac_speech/jac_speech/vc_tts/README.md:20]: sudo apt-get install -y espeak-ng

**Review by chandralegend — CHANGES_REQUESTED** [review]

**Review by AshishMahendra — APPROVED** [review]
LGTM

**Pull request merged**